source/manual/alerts/root-filesystem-is-readonly.html.md

```diff
@@ -13,9 +13,15 @@ read only. This could be caused by a cloud hosting provider having
 problems with their Storage Area Network (SAN).
 
 The machine will need to have a filesystem check forced on reboot. To do
-this log into the vCloud Director instance for the environment, find the
-affected machine and click on the 'console' image to bring up a web
-based console. Click the 'reset' icon in the top bar and then click to
-focus on the console. Press the 'f' key when prompted (or repeatedly if
-there's no output) to force a filesystem check.
+this you will need access to the vCloud Director instance for the 
+environment, this access is typically only available to members
+of the infrastructure team.
```

Comment:
"...of the GOVUK Infrastructure team. However, if you do on-call, you should have access to the vCloud Production environment."